modules/ROOT/pages/storage.adoc

```diff
@@ -0,0 +1,87 @@
+= Configuring Storage
+
+Fedora CoreOS ships with a simple default storage layout: the root partition is the last one and expands to take the full size of the disk. Apart from the boot partition, all data is stored on the root partition.
+
+Below, we provide examples of various ways you can customize this.
+
+== Setting up separate /var mounts
+
+Here's an example FCC file to set up `/var` on a separate partition on the same primary disk:
+
+.Adding a /var partition to the primary disk
+[source,yaml]
+----
+variant: fcos
+version: 1.1.0
+storage:
+  disks:
+  - # The name of the primary block device. In virtio-based setups, this is
+    # likely `/dev/vda`. Elsewhere, it's likely `/dev/sda`.
+    device: /dev/vda
+    # We do not want to wipe the partition table since this is the primary
+    # device.
+    wipe_table: false
+    partitions:
+    - size_mib: 0
+      # Start at 5G so that we leave enough space for the root partition.
```

Comment:
I would duplicate those comments in the config below to make sure someone only looking at the one they need would not miss them.

Reply:
Good idea, thanks!